feat: add dispatch core mfa plugin for signal engagement

wssheldon opened the pull request:
Adds a plugin for signal engagement confirmation where Dispatch authentication is the source of additional verification for an engaged user.

Commit:
feat: add dispatch core mfa plugin for signal engagement

kevgliss commented on src/dispatch/plugins/dispatch_core/plugin.py:
```diff
@@ -273,6 +279,102 @@ def get(
+        start_time = time.time()
+
+        while time.time() - start_time < timeout:
+            with get_organization_session(organization_slug="default") as session:
```
If this is within a web request shouldn't it already be scoped to the organization?

kevgliss commented on src/dispatch/plugins/dispatch_core/plugin.py:
```diff
@@ -273,6 +279,102 @@ def get(
+            time.sleep(1)
+
+        # Update the challenge status to EXPIRED if it times out
+        with get_organization_session(organization_slug="default") as session:
```
Same question as above.

kevgliss commented on src/dispatch/plugins/dispatch_core/plugin.py:
```diff
@@ -273,6 +279,102 @@ def get(
+        """Validates a multi-factor authentication token."""
+        print(f"got here: {payload.challenge_id=} {payload.__dict__=}")
+        challenge = db_session.query(MfaChallenge).filter_by(challenge_id=payload.challenge_id).one_or_none()
+        print(f"disp {challenge.dispatch_user_id=} and {payload.user_id=} {type(challenge.dispatch_user_id)=} {type(payload.user_id)=}")
```
Remove or move to logging.

kevgliss commented on src/dispatch/static/dispatch/src/auth/Mfa.vue:
```diff
@@ -0,0 +1,56 @@
+      challenge_id: challengeId,
+      project_id: projectId,
+      action: action,
+      user_id: 12480, // TODO(whsel): fetch this
```
I think this should be done server side.

whitdog47 committed:
disables tab in incident edit until populated (#5075)

dependabot[bot] committed:
chore(deps): bump @tiptap/pm in /src/dispatch/static/dispatch (#5079)

Bumps [@tiptap/pm](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/pm) from 2.5.9 to 2.6.2.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/develop/packages/pm/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/pm@2.6.2/packages/pm)

---
updated-dependencies:
- dependency-name: "@tiptap/pm"
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/extension-placeholder (#5078)

Bumps [@tiptap/extension-placeholder](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/extension-placeholder) from 2.5.9 to 2.6.2.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/develop/packages/extension-placeholder/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/extension-placeholder@2.6.2/packages/extension-placeholder)

---
updated-dependencies:
- dependency-name: "@tiptap/extension-placeholder"
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/vue-3 in /src/dispatch/static/dispatch (#5077)

Bumps [@tiptap/vue-3](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/vue-3) from 2.5.9 to 2.6.2.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/develop/packages/vue-3/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/vue-3@2.6.2/packages/vue-3)

---
updated-dependencies:
- dependency-name: "@tiptap/vue-3"
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/starter-kit in /src/dispatch/static/dispatch (#5076)

Bumps [@tiptap/starter-kit](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/starter-kit) from 2.5.9 to 2.6.2.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/develop/packages/starter-kit/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/starter-kit@2.6.2/packages/starter-kit)

---
updated-dependencies:
- dependency-name: "@tiptap/starter-kit"
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump faker from 26.3.0 to 27.0.0 (#5072)

Bumps [faker](https://github.com/joke2k/faker) from 26.3.0 to 27.0.0.
- [Release notes](https://github.com/joke2k/faker/releases)
- [Changelog](https://github.com/joke2k/faker/blob/master/CHANGELOG.md)
- [Commits](https://github.com/joke2k/faker/compare/v26.3.0...v27.0.0)

---
updated-dependencies:
- dependency-name: faker
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump uvicorn from 0.30.5 to 0.30.6 (#5071)

Bumps [uvicorn](https://github.com/encode/uvicorn) from 0.30.5 to 0.30.6.
- [Release notes](https://github.com/encode/uvicorn/releases)
- [Changelog](https://github.com/encode/uvicorn/blob/master/CHANGELOG.md)
- [Commits](https://github.com/encode/uvicorn/compare/0.30.5...0.30.6)

---
updated-dependencies:
- dependency-name: uvicorn
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/starter-kit in /src/dispatch/static/dispatch (#5080)

Bumps [@tiptap/starter-kit](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/starter-kit) from 2.6.2 to 2.6.3.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/develop/packages/starter-kit/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/starter-kit@2.6.3/packages/starter-kit)

---
updated-dependencies:
- dependency-name: "@tiptap/starter-kit"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump vuetify in /src/dispatch/static/dispatch (#5091)

Bumps [vuetify](https://github.com/vuetifyjs/vuetify/tree/HEAD/packages/vuetify) from 3.6.14 to 3.7.0.
- [Release notes](https://github.com/vuetifyjs/vuetify/releases)
- [Commits](https://github.com/vuetifyjs/vuetify/commits/v3.7.0/packages/vuetify)

---
updated-dependencies:
- dependency-name: vuetify
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump uvloop from 0.19.0 to 0.20.0 (#5090)

Bumps [uvloop](https://github.com/MagicStack/uvloop) from 0.19.0 to 0.20.0.
- [Release notes](https://github.com/MagicStack/uvloop/releases)
- [Commits](https://github.com/MagicStack/uvloop/compare/v0.19.0...v0.20.0)

---
updated-dependencies:
- dependency-name: uvloop
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump ruff from 0.5.7 to 0.6.0 (#5089)

Bumps [ruff](https://github.com/astral-sh/ruff) from 0.5.7 to 0.6.0.
- [Release notes](https://github.com/astral-sh/ruff/releases)
- [Changelog](https://github.com/astral-sh/ruff/blob/main/CHANGELOG.md)
- [Commits](https://github.com/astral-sh/ruff/compare/0.5.7...0.6.0)

---
updated-dependencies:
- dependency-name: ruff
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump slack-bolt from 1.19.1 to 1.20.0 (#5087)

Bumps [slack-bolt](https://github.com/slackapi/bolt-python) from 1.19.1 to 1.20.0.
- [Release notes](https://github.com/slackapi/bolt-python/releases)
- [Commits](https://github.com/slackapi/bolt-python/compare/v1.19.1...v1.20.0)

---
updated-dependencies:
- dependency-name: slack-bolt
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump google-api-python-client from 2.140.0 to 2.141.0 (#5085)

Bumps [google-api-python-client](https://github.com/googleapis/google-api-python-client) from 2.140.0 to 2.141.0.
- [Release notes](https://github.com/googleapis/google-api-python-client/releases)
- [Commits](https://github.com/googleapis/google-api-python-client/compare/v2.140.0...v2.141.0)

---
updated-dependencies:
- dependency-name: google-api-python-client
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/extension-placeholder (#5084)

Bumps [@tiptap/extension-placeholder](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/extension-placeholder) from 2.6.2 to 2.6.4.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/develop/packages/extension-placeholder/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/extension-placeholder@2.6.4/packages/extension-placeholder)

---
updated-dependencies:
- dependency-name: "@tiptap/extension-placeholder"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/pm in /src/dispatch/static/dispatch (#5083)

Bumps [@tiptap/pm](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/pm) from 2.6.3 to 2.6.4.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/develop/packages/pm/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/pm@2.6.4/packages/pm)

---
updated-dependencies:
- dependency-name: "@tiptap/pm"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/vue-3 in /src/dispatch/static/dispatch (#5082)

Bumps [@tiptap/vue-3](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/vue-3) from 2.6.2 to 2.6.4.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/develop/packages/vue-3/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/vue-3@2.6.4/packages/vue-3)

---
updated-dependencies:
- dependency-name: "@tiptap/vue-3"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump vue in /src/dispatch/static/dispatch (#5081)

Bumps [vue](https://github.com/vuejs/core) from 3.4.37 to 3.4.38.
- [Release notes](https://github.com/vuejs/core/releases)
- [Changelog](https://github.com/vuejs/core/blob/v3.4.38/CHANGELOG.md)
- [Commits](https://github.com/vuejs/core/compare/v3.4.37...v3.4.38)

---
updated-dependencies:
- dependency-name: vue
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tanstack/vue-query in /src/dispatch/static/dispatch (#5102)

Bumps [@tanstack/vue-query](https://github.com/TanStack/query/tree/HEAD/packages/vue-query) from 5.51.21 to 5.51.24.
- [Release notes](https://github.com/TanStack/query/releases)
- [Commits](https://github.com/TanStack/query/commits/v5.51.24/packages/vue-query)

---
updated-dependencies:
- dependency-name: "@tanstack/vue-query"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/starter-kit in /src/dispatch/static/dispatch (#5101)

Bumps [@tiptap/starter-kit](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/starter-kit) from 2.6.3 to 2.6.4.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/develop/packages/starter-kit/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/starter-kit@2.6.4/packages/starter-kit)

---
updated-dependencies:
- dependency-name: "@tiptap/starter-kit"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump vite in /src/dispatch/static/dispatch (#5100)

Bumps [vite](https://github.com/vitejs/vite/tree/HEAD/packages/vite) from 5.4.0 to 5.4.1.
- [Release notes](https://github.com/vitejs/vite/releases)
- [Changelog](https://github.com/vitejs/vite/blob/main/packages/vite/CHANGELOG.md)
- [Commits](https://github.com/vitejs/vite/commits/v5.4.1/packages/vite)

---
updated-dependencies:
- dependency-name: vite
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump @playwright/test in /src/dispatch/static/dispatch (#5099)

Bumps [@playwright/test](https://github.com/microsoft/playwright) from 1.46.0 to 1.46.1.
- [Release notes](https://github.com/microsoft/playwright/releases)
- [Commits](https://github.com/microsoft/playwright/compare/v1.46.0...v1.46.1)

---
updated-dependencies:
- dependency-name: "@playwright/test"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump cachetools from 5.4.0 to 5.5.0 (#5098)

Bumps [cachetools](https://github.com/tkem/cachetools) from 5.4.0 to 5.5.0.
- [Changelog](https://github.com/tkem/cachetools/blob/master/CHANGELOG.rst)
- [Commits](https://github.com/tkem/cachetools/compare/v5.4.0...v5.5.0)

---
updated-dependencies:
- dependency-name: cachetools
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump aiohttp from 3.10.3 to 3.10.4 (#5097)

Bumps [aiohttp](https://github.com/aio-libs/aiohttp) from 3.10.3 to 3.10.4.
- [Release notes](https://github.com/aio-libs/aiohttp/releases)
- [Changelog](https://github.com/aio-libs/aiohttp/blob/master/CHANGES.rst)
- [Commits](https://github.com/aio-libs/aiohttp/compare/v3.10.3...v3.10.4)

---
updated-dependencies:
- dependency-name: aiohttp
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump factory-boy from 3.3.0 to 3.3.1 (#5096)

Bumps [factory-boy](https://github.com/FactoryBoy/factory_boy) from 3.3.0 to 3.3.1.
- [Changelog](https://github.com/FactoryBoy/factory_boy/blob/master/docs/changelog.rst)
- [Commits](https://github.com/FactoryBoy/factory_boy/compare/3.3.0...3.3.1)

---
updated-dependencies:
- dependency-name: factory-boy
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump ruff from 0.6.0 to 0.6.1 (#5095)

Bumps [ruff](https://github.com/astral-sh/ruff) from 0.6.0 to 0.6.1.
- [Release notes](https://github.com/astral-sh/ruff/releases)
- [Changelog](https://github.com/astral-sh/ruff/blob/main/CHANGELOG.md)
- [Commits](https://github.com/astral-sh/ruff/compare/0.6.0...0.6.1)

---
updated-dependencies:
- dependency-name: ruff
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump markdown from 3.6 to 3.7 (#5094)

Bumps [markdown](https://github.com/Python-Markdown/markdown) from 3.6 to 3.7.
- [Release notes](https://github.com/Python-Markdown/markdown/releases)
- [Changelog](https://github.com/Python-Markdown/markdown/blob/master/docs/changelog.md)
- [Commits](https://github.com/Python-Markdown/markdown/compare/3.6...3.7)

---
updated-dependencies:
- dependency-name: markdown
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

whitdog47 committed:
Allow scribes to create a timeline event (#5092)

dependabot[bot] committed:
chore(deps): bump aiohttp from 3.10.4 to 3.10.5 (#5104)

Bumps [aiohttp](https://github.com/aio-libs/aiohttp) from 3.10.4 to 3.10.5.
- [Release notes](https://github.com/aio-libs/aiohttp/releases)
- [Changelog](https://github.com/aio-libs/aiohttp/blob/master/CHANGES.rst)
- [Commits](https://github.com/aio-libs/aiohttp/compare/v3.10.4...v3.10.5)

---
updated-dependencies:
- dependency-name: aiohttp
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @vue-flow/core in /src/dispatch/static/dispatch (#5103)

Bumps [@vue-flow/core](https://github.com/bcakmakoglu/vue-flow) from 1.39.3 to 1.40.1.
- [Release notes](https://github.com/bcakmakoglu/vue-flow/releases)
- [Changelog](https://github.com/bcakmakoglu/vue-flow/blob/master/CHANGELOG.md)
- [Commits](https://github.com/bcakmakoglu/vue-flow/compare/@vue-flow/core@1.39.3...v1.40.1)

---
updated-dependencies:
- dependency-name: "@vue-flow/core"
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

whitdog47 committed:
Removes subqueries to ensure pagination works (#5105)

whitdog47 committed:
Adding handler for join case from case notification (#5088)

dependabot[bot] committed:
chore(deps): bump spacy from 3.7.5 to 3.7.6 (#5112)

Bumps [spacy](https://github.com/explosion/spaCy) from 3.7.5 to 3.7.6.
- [Release notes](https://github.com/explosion/spaCy/releases)
- [Commits](https://github.com/explosion/spaCy/compare/v3.7.5...release-v3.7.6)

---
updated-dependencies:
- dependency-name: spacy
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump google-api-python-client from 2.141.0 to 2.142.0 (#5111)

Bumps [google-api-python-client](https://github.com/googleapis/google-api-python-client) from 2.141.0 to 2.142.0.
- [Release notes](https://github.com/googleapis/google-api-python-client/releases)
- [Commits](https://github.com/googleapis/google-api-python-client/compare/v2.141.0...v2.142.0)

---
updated-dependencies:
- dependency-name: google-api-python-client
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/vue-3 in /src/dispatch/static/dispatch (#5110)

Bumps [@tiptap/vue-3](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/vue-3) from 2.6.4 to 2.6.5.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/develop/packages/vue-3/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/vue-3@2.6.5/packages/vue-3)

---
updated-dependencies:
- dependency-name: "@tiptap/vue-3"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump vite in /src/dispatch/static/dispatch (#5109)

Bumps [vite](https://github.com/vitejs/vite/tree/HEAD/packages/vite) from 5.4.1 to 5.4.2.
- [Release notes](https://github.com/vitejs/vite/releases)
- [Changelog](https://github.com/vitejs/vite/blob/main/packages/vite/CHANGELOG.md)
- [Commits](https://github.com/vitejs/vite/commits/v5.4.2/packages/vite)

---
updated-dependencies:
- dependency-name: vite
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/extension-placeholder (#5108)

Bumps [@tiptap/extension-placeholder](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/extension-placeholder) from 2.6.4 to 2.6.5.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/develop/packages/extension-placeholder/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/extension-placeholder@2.6.5/packages/extension-placeholder)

---
updated-dependencies:
- dependency-name: "@tiptap/extension-placeholder"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/starter-kit in /src/dispatch/static/dispatch (#5107)

Bumps [@tiptap/starter-kit](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/starter-kit) from 2.6.4 to 2.6.5.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/develop/packages/starter-kit/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/starter-kit@2.6.5/packages/starter-kit)

---
updated-dependencies:
- dependency-name: "@tiptap/starter-kit"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tanstack/vue-query in /src/dispatch/static/dispatch (#5106)

Bumps [@tanstack/vue-query](https://github.com/TanStack/query/tree/HEAD/packages/vue-query) from 5.51.24 to 5.52.0.
- [Release notes](https://github.com/TanStack/query/releases)
- [Commits](https://github.com/TanStack/query/commits/v5.52.0/packages/vue-query)

---
updated-dependencies:
- dependency-name: "@tanstack/vue-query"
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

whitdog47 committed:
Truncate the case resolution reason text due to Slack limit (#5093)

whitdog47 committed:
Modifying max limit (#5115)

kevgliss committed:
Adding some better error handling to sqs consume (#5113)

Co-authored-by: Will Sheldon <114631109+wssheldon@users.noreply.github.com>

kevgliss committed:
Refact(signals)/create instance (#5114)

* Removing the ability to update signals based on uuid

* Add todo

* Removing unused import

* Removing unused imports

---------

Co-authored-by: David Whittaker <dwhittaker@netflix.com>
Co-authored-by: Will Sheldon <114631109+wssheldon@users.noreply.github.com>

Commit:
enhancement(cli/consume): improve thread exception handling in cli consumer (#5118)

* enhancement(cli/consume): improve thread exception handling in cli consumer

* enhancement(cli/consume): improve thread exception handling in cli consumer

Commit:
refactor(plugins/aws): error handling, logging, types, and early returns (#5117)

* refactor(plugins/aws): error handling, logging, types, and early returns

* logs: add specific log message to create instance exception

ashleytolbert committed:
Adding project setting to allow visibility selection of incident command (#5129)

* Adding new attribute to project model for commander setting visibility

* Adding changes to make visibility of the specify-incident-commander in the report incident form be selectable by project

* Adding changes for incident commander selection visibility

* Update src/dispatch/static/dispatch/src/incident/ReportSubmissionCard.vue

Fixes JS errors in ReportSubmissionCard.vue

Co-authored-by: David Whittaker <84562015+whitdog47@users.noreply.github.com>

* Update ReportSubmissionCard.vue

Fixes JS erros

* Update NewEditSheet.vue

Fixes JS syntax error

* Update store.js

Fixes JS error

---------

Co-authored-by: Ashley Tolbert <artolbert@neflix.com>
Co-authored-by: David Whittaker <84562015+whitdog47@users.noreply.github.com>

metroid-samus committed:
Enable backend functionality of calculating case costs. (#4900)

* Add cost models to case types

* Adds case cost type

* Create case costs

* Make db changes

* add cases to participant activity

* Makes case field in participant activity optional

* Fixes lint errors

* Fixes sqlalchemy upgrade/downgrade.

* Fixes lint errors.

* Adds case cost type service

* Enable case participant activity updates.

* Renames function to be inclusive of cases

* Gets all open cases by case type.

* Calculates case costs.

* Remove wrongly added file.

* Restore previous case type service

* Adds case cost scheduler

* Enable adding/removal of case cost items.

* Remove unused code.

* Check for escalated cases as well.

* Update src/dispatch/participant_activity/service.py

Co-authored-by: Will Sheldon <114631109+wssheldon@users.noreply.github.com>

* Fix import

* Load only relevant columns when calculating case costs

* Load case costs

* Do not add relationships

* Reduce the number of Case fields loaded during scheduled cost updates

* Retrieve new and triaged cases.

* Enable users to select cases based on a list of statuses

* Fixing tests

---------

Co-authored-by: Will Sheldon <114631109+wssheldon@users.noreply.github.com>
Co-authored-by: David Whittaker <dwhittaker@netflix.com>

Commit:
fix(plugins): aws consume plugin transaction management (#5131)

* fix(plugins): aws consume plugin transaction management

* fix(plugins): aws consume plugin transaction management

* fix(plugins): aws consume plugin transaction management

mvilanova committed:
Asks user to type database hostname and name before asking for confirmation to drop database (#5132)

Commit:
fix(plugin/aws): catch integrity error and log warning (#5133)

dependabot[bot] committed:
chore(deps): bump @tanstack/vue-query in /src/dispatch/static/dispatch (#5130)

Bumps [@tanstack/vue-query](https://github.com/TanStack/query/tree/HEAD/packages/vue-query) from 5.52.0 to 5.52.2.
- [Release notes](https://github.com/TanStack/query/releases)
- [Commits](https://github.com/TanStack/query/commits/v5.52.2/packages/vue-query)

---
updated-dependencies:
- dependency-name: "@tanstack/vue-query"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump faker from 27.0.0 to 28.0.0 (#5127)

Bumps [faker](https://github.com/joke2k/faker) from 27.0.0 to 28.0.0.
- [Release notes](https://github.com/joke2k/faker/releases)
- [Changelog](https://github.com/joke2k/faker/blob/master/CHANGELOG.md)
- [Commits](https://github.com/joke2k/faker/compare/v27.0.0...v28.0.0)

---
updated-dependencies:
- dependency-name: faker
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump pyparsing from 3.1.2 to 3.1.4 (#5126)

Bumps [pyparsing](https://github.com/pyparsing/pyparsing) from 3.1.2 to 3.1.4.
- [Release notes](https://github.com/pyparsing/pyparsing/releases)
- [Changelog](https://github.com/pyparsing/pyparsing/blob/master/CHANGES)
- [Commits](https://github.com/pyparsing/pyparsing/compare/pyparsing_3.1.2...3.1.4)

---
updated-dependencies:
- dependency-name: pyparsing
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump slack-bolt from 1.20.0 to 1.20.1 (#5125)

Bumps [slack-bolt](https://github.com/slackapi/bolt-python) from 1.20.0 to 1.20.1.
- [Release notes](https://github.com/slackapi/bolt-python/releases)
- [Commits](https://github.com/slackapi/bolt-python/compare/v1.20.0...v1.20.1)

---
updated-dependencies:
- dependency-name: slack-bolt
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump ruff from 0.6.1 to 0.6.2 (#5124)

Bumps [ruff](https://github.com/astral-sh/ruff) from 0.6.1 to 0.6.2.
- [Release notes](https://github.com/astral-sh/ruff/releases)
- [Changelog](https://github.com/astral-sh/ruff/blob/main/CHANGELOG.md)
- [Commits](https://github.com/astral-sh/ruff/compare/0.6.1...0.6.2)

---
updated-dependencies:
- dependency-name: ruff
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/extension-placeholder (#5123)

Bumps [@tiptap/extension-placeholder](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/extension-placeholder) from 2.6.5 to 2.6.6.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/@tiptap/extension-placeholder@2.6.6/packages/extension-placeholder/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/extension-placeholder@2.6.6/packages/extension-placeholder)

---
updated-dependencies:
- dependency-name: "@tiptap/extension-placeholder"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/vue-3 in /src/dispatch/static/dispatch (#5122)

Bumps [@tiptap/vue-3](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/vue-3) from 2.6.5 to 2.6.6.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/@tiptap/vue-3@2.6.6/packages/vue-3/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/vue-3@2.6.6/packages/vue-3)

---
updated-dependencies:
- dependency-name: "@tiptap/vue-3"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/starter-kit in /src/dispatch/static/dispatch (#5121)

Bumps [@tiptap/starter-kit](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/starter-kit) from 2.6.5 to 2.6.6.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/@tiptap/starter-kit@2.6.6/packages/starter-kit/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/starter-kit@2.6.6/packages/starter-kit)

---
updated-dependencies:
- dependency-name: "@tiptap/starter-kit"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @tiptap/pm in /src/dispatch/static/dispatch (#5120)

Bumps [@tiptap/pm](https://github.com/ueberdosis/tiptap/tree/HEAD/packages/pm) from 2.6.5 to 2.6.6.
- [Release notes](https://github.com/ueberdosis/tiptap/releases)
- [Changelog](https://github.com/ueberdosis/tiptap/blob/@tiptap/pm@2.6.6/packages/pm/CHANGELOG.md)
- [Commits](https://github.com/ueberdosis/tiptap/commits/@tiptap/pm@2.6.6/packages/pm)

---
updated-dependencies:
- dependency-name: "@tiptap/pm"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump pydantic from 1.10.17 to 1.10.18 (#5119)

Bumps [pydantic](https://github.com/pydantic/pydantic) from 1.10.17 to 1.10.18.
- [Release notes](https://github.com/pydantic/pydantic/releases)
- [Changelog](https://github.com/pydantic/pydantic/blob/main/HISTORY.md)
- [Commits](https://github.com/pydantic/pydantic/compare/v1.10.17...v1.10.18)

---
updated-dependencies:
- dependency-name: pydantic
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump webpack from 5.77.0 to 5.94.0 in /docs (#5137)

Bumps [webpack](https://github.com/webpack/webpack) from 5.77.0 to 5.94.0.
- [Release notes](https://github.com/webpack/webpack/releases)
- [Commits](https://github.com/webpack/webpack/compare/v5.77.0...v5.94.0)

---
updated-dependencies:
- dependency-name: webpack
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump micromatch from 4.0.5 to 4.0.8 in /docs (#5136)

Bumps [micromatch](https://github.com/micromatch/micromatch) from 4.0.5 to 4.0.8.
- [Release notes](https://github.com/micromatch/micromatch/releases)
- [Changelog](https://github.com/micromatch/micromatch/blob/master/CHANGELOG.md)
- [Commits](https://github.com/micromatch/micromatch/compare/4.0.5...4.0.8)

---
updated-dependencies:
- dependency-name: micromatch
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump micromatch in /src/dispatch/static/dispatch (#5135)

Bumps [micromatch](https://github.com/micromatch/micromatch) from 4.0.5 to 4.0.8.
- [Release notes](https://github.com/micromatch/micromatch/releases)
- [Changelog](https://github.com/micromatch/micromatch/blob/master/CHANGELOG.md)
- [Commits](https://github.com/micromatch/micromatch/compare/4.0.5...4.0.8)

---
updated-dependencies:
- dependency-name: micromatch
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

metroid-samus committed:
Delete unused code and fix logging. (#5138)

metroid-samus committed:
Fixes plugin instance call to fetch events. (#5139)

Commit:
refactor(plugin/aws): single threaded consume, call rollback (#5144)

* refactor(plugin/aws): single threaded consume, call rollback

* revert: rebase drop_database

* revert: rebase drop_database

metroid-samus committed:
Case cost model - frontend (#5140)

* Add cost models to case types

* Fixes lint errors

* Adds case cost type service

* Enable case participant activity updates.

* Renames function to be inclusive of cases

* Gets all open cases by case type.

* Calculates case costs.

* Adds case cost scheduler

* Adds UI for case cost type and adds case cost types api calls.

* Merged

* Adds case cost pages.

* Enable users to set case cost models based on case type.

* Reduce diff.

* Fixes javascript lint errors.

* Reduce the frequency of aggregating case costs.

* Remove extra cost model combobox

* Update src/dispatch/case_cost/views.py

Co-authored-by: Will Sheldon <114631109+wssheldon@users.noreply.github.com>

* Update src/dispatch/case_cost/views.py

Co-authored-by: Marc Vilanova <39573146+mvilanova@users.noreply.github.com>

* Fixes grammar.

---------

Co-authored-by: Will Sheldon <114631109+wssheldon@users.noreply.github.com>
Co-authored-by: Marc Vilanova <39573146+mvilanova@users.noreply.github.com>

whitdog47 committed:
Check role before reassigning (#5152)

dependabot[bot] committed:
chore(deps-dev): bump faker from 28.0.0 to 28.1.0 (#5154)

Bumps [faker](https://github.com/joke2k/faker) from 28.0.0 to 28.1.0.
- [Release notes](https://github.com/joke2k/faker/releases)
- [Changelog](https://github.com/joke2k/faker/blob/master/CHANGELOG.md)
- [Commits](https://github.com/joke2k/faker/compare/v28.0.0...v28.1.0)

---
updated-dependencies:
- dependency-name: faker
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump ipython from 8.26.0 to 8.27.0 (#5151)

Bumps [ipython](https://github.com/ipython/ipython) from 8.26.0 to 8.27.0.
- [Release notes](https://github.com/ipython/ipython/releases)
- [Commits](https://github.com/ipython/ipython/compare/8.26.0...8.27.0)

---
updated-dependencies:
- dependency-name: ipython
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump ruff from 0.6.2 to 0.6.3 (#5150)

Bumps [ruff](https://github.com/astral-sh/ruff) from 0.6.2 to 0.6.3.
- [Release notes](https://github.com/astral-sh/ruff/releases)
- [Changelog](https://github.com/astral-sh/ruff/blob/main/CHANGELOG.md)
- [Commits](https://github.com/astral-sh/ruff/compare/0.6.2...0.6.3)

---
updated-dependencies:
- dependency-name: ruff
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump apexcharts in /src/dispatch/static/dispatch (#5149)

Bumps [apexcharts](https://github.com/apexcharts/apexcharts.js) from 3.52.0 to 3.53.0.
- [Release notes](https://github.com/apexcharts/apexcharts.js/releases)
- [Commits](https://github.com/apexcharts/apexcharts.js/commits)

---
updated-dependencies:
- dependency-name: apexcharts
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump @vue-flow/core in /src/dispatch/static/dispatch (#5148)

Bumps [@vue-flow/core](https://github.com/bcakmakoglu/vue-flow) from 1.40.1 to 1.41.0.
- [Release notes](https://github.com/bcakmakoglu/vue-flow/releases)
- [Changelog](https://github.com/bcakmakoglu/vue-flow/blob/master/CHANGELOG.md)
- [Commits](https://github.com/bcakmakoglu/vue-flow/compare/@vue-flow/core@1.40.1...v1.41.0)

---
updated-dependencies:
- dependency-name: "@vue-flow/core"
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump vue3-apexcharts in /src/dispatch/static/dispatch (#5147)

Bumps [vue3-apexcharts](https://github.com/apexcharts/vue3-apexcharts) from 1.5.3 to 1.6.0.
- [Commits](https://github.com/apexcharts/vue3-apexcharts/commits)

---
updated-dependencies:
- dependency-name: vue3-apexcharts
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump @vitejs/plugin-vue (#5146)

Bumps [@vitejs/plugin-vue](https://github.com/vitejs/vite-plugin-vue/tree/HEAD/packages/plugin-vue) from 5.1.2 to 5.1.3.
- [Release notes](https://github.com/vitejs/vite-plugin-vue/releases)
- [Changelog](https://github.com/vitejs/vite-plugin-vue/blob/main/packages/plugin-vue/CHANGELOG.md)
- [Commits](https://github.com/vitejs/vite-plugin-vue/commits/plugin-vue@5.1.3/packages/plugin-vue)

---
updated-dependencies:
- dependency-name: "@vitejs/plugin-vue"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump httpx from 0.27.0 to 0.27.2 (#5143)

Bumps [httpx](https://github.com/encode/httpx) from 0.27.0 to 0.27.2.
- [Release notes](https://github.com/encode/httpx/releases)
- [Changelog](https://github.com/encode/httpx/blob/master/CHANGELOG.md)
- [Commits](https://github.com/encode/httpx/compare/0.27.0...0.27.2)

---
updated-dependencies:
- dependency-name: httpx
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump google-api-python-client from 2.142.0 to 2.143.0 (#5142)

Bumps [google-api-python-client](https://github.com/googleapis/google-api-python-client) from 2.142.0 to 2.143.0.
- [Release notes](https://github.com/googleapis/google-api-python-client/releases)
- [Commits](https://github.com/googleapis/google-api-python-client/compare/v2.142.0...v2.143.0)

---
updated-dependencies:
- dependency-name: google-api-python-client
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump vuetify in /src/dispatch/static/dispatch (#5141)

Bumps [vuetify](https://github.com/vuetifyjs/vuetify/tree/HEAD/packages/vuetify) from 3.7.0 to 3.7.1.
- [Release notes](https://github.com/vuetifyjs/vuetify/releases)
- [Commits](https://github.com/vuetifyjs/vuetify/commits/v3.7.1/packages/vuetify)

---
updated-dependencies:
- dependency-name: vuetify
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Avery <averyl@netflix.com>

metroid-samus committed:
Case cost model is an experimental feature. (#5156)

whitdog47 committed:
Fixing participant role type in mapping function (#5157)

metroid-samus committed:
Hide cost column from Case table if experimental features are disabled. (#5158)

* Hide cost column from Case table if experimental features are disabled.

* Fixes formatting.

whitdog47 committed:
Fixing role formatting for timeline event (#5159)

whitdog47 committed:
Fixes selection filtering to project (#5166)

wssheldon commented:
Bumping..

Commit:
feat(plugins/aws): check for dupe instance id (#5174)

Commit:
feat(plugin/mfa): redesign UI, update slack msg, fix session usage
